config/taxonomy/codegen.spec.yaml

```diff
@@ -8,26 +8,22 @@ servers:
   - url: https://localhost:8080
 paths:
   /getPoliciesDecisions:
-    get:
-      summary: getPoliciesDecisions
-      description: Rest API for returning policy decisions
-      operationId: getPoliciesDecisions
+    post:
+      summary: getPoliciesDecisions.
       parameters:
-        - name: input
-          description: Policy Manager Request Object
-          required: true
-          in: query
-          content:
-            application/json:
-              schema:
-                type: object
-                $ref: "./objects/policy_manager_request.json"
-        - name: creds
-          description: credentials to wkc connector
-          required: true
-          in: query
+        - in: header
+          name: X-Request-Cred
           schema:
             type: string
+          required: true
+      requestBody:
+        description: Policy Manager Request Object.
+        required: true
+        content:
+          application/json: 
+            schema:
+              type: object
```

Comment:
Is this needed? From what I know if there is `$ref` then there can be no siblings